boards/xtensa/esp32/esp32-wrover-kit/configs/lvgl/defconfig

```diff
@@ -42,7 +42,13 @@ CONFIG_LCD_DEV=y
 CONFIG_LCD_ILI9341=y
 CONFIG_LCD_ILI9341_IFACE0=y
 CONFIG_LV_COLOR_16_SWAP=y
-CONFIG_LV_DPI=125
+CONFIG_LV_MEM_CUSTOM=y
+CONFIG_LV_PORT_USE_LCDDEV=y
+CONFIG_LV_PORT_USE_TOUCHPAD=y
```

Comment:
ESP-WROVER-KIT does not provide input support with LVGL.

Reply:
Sure, I am testing it right now.
But so far `lvgldemo widgets` is failing to run:
```
nsh> lvgldemo widgets
[LVGL] [Warn]   (5.340, +5340)   lv_demo_widgets: LV_FONT_MONTSERRAT_18 is not enabled for the widgets demo. Using LV_FONT_DEFAULT instead.     (in lv_demo_widgets.c line #138)
[LVGL] [Warn]   (5.340, +0)      lv_demo_widgets: LV_FONT_MONTSERRAT_12 is not enabled for the widgets demo. Using LV_FONT_DEFAULT instead.     (in lv_demo_widgets.c line #143)
[LVGL] [Error]  (5.860, +520)    lcddev_update_thread: ioctl(LCDDEVIO_PUTAREA) failed: 22       (in lv_port_lcddev.c line #104)
```

Reply:
With the patch from #7277, I managed to execute the Widgets demo on the ESP-WROVER-KIT development board.

Reply:
`CONFIG_LV_PORT_USE_TOUCHPAD` still needs to be removed from the defconfig.